source/lambda/kendraDataSourceCreator/lambda_function.py

```diff
@@ -0,0 +1,41 @@
+import boto3
```

Comment:
Should we rename the file to CustomResourceKendraIndex?
Since it does take care of creation as well as other events?

Reply:
Shouldn't the lambda functions be named as verbs?

Reply:
* Functions need to be named as verbs. Though we are using lambda functions, we are using a lambda more as a "module" which consists of multiple functionalities. The functionalities have verbs (such as update,create,delete).
The current name implies that this lambda only takes care of creation.
Also, the current name "kendraDataSourceCreator" is a noun and not a verb

Reply:
Will change it.